### PR TITLE
[DX-511] UDG overview page should be replaced.

### PR DESCRIPTION
**Pull request by yurisasuke**
### For internal users - please add a Jira DX PR ticket to the subject!
---
DX-511
### Preview Link
https://deploy-preview-2964--tyk-docs.netlify.app/docs/nightly/universal-data-graph/

### Description
UDG overview page should be replaced with /universal-data-graph.

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/data/menu.yaml [hunk 1]
```diff
@@ -2616,6 +2612,10 @@ menu:
         category: Directory
         show: True
         menu:
+        - title: "GraphQl Datasource"
```
Comment (dcs3spp) [inline tyk-docs/data/menu.yaml]: Make the title "GraphQL Datasource"

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Hi @Keithwachira, PR looks good with overview page updated.

For this path /universal-data-graph/datasources/graphql update the title from "GraphQl Data source" to "GraphQL Datasource"

File: tyk-docs/data/menu.yaml (hunk 1 above)
Comment (yurisasuke) [reply tyk-docs/data/menu.yaml]: fixed

**Review by dcs3spp — APPROVED** [review]
lgtm

**Pull request merged**